Makefile

```diff
@@ -127,28 +135,10 @@ clean:
 	cd images && $(MAKE) clean
 	rm -rf wd_suite
 
-$(GOMETALINTER):
-	go get -u gopkg.in/alecthomas/gometalinter.v2
-	ln -s $(GOPATH)/bin/gometalinter.v2 $(GOPATH)/bin/gometalinter
-	gometalinter --install
-
-.PHONY: lint
-lint: $(GOMETALINTER)
-	gometalinter --vendor --skip images/stolon/stolon --disable-all \
-		--enable=deadcode \
-		--enable=ineffassign \
-		--enable=gosimple \
-		--enable=staticcheck \
-		--enable=gofmt \
-		--enable=goimports \
-		--enable=dupl \
-		--enable=misspell \
-		--enable=errcheck \
-		--enable=vet \
-		--enable=vetshadow \
-		--deadline=1m \
-		./...
-
 .PHONY: fix-logrus
 fix-logrus:
 	find vendor -type f -print0 | xargs -0 sed -i 's/Sirupsen/sirupsen/g'
+
+.PHONY: lint
```

Comment:
nit: I think lint should be as a prerequisite to an (image) build or some artifact generating step.